### PR TITLE
Help Button Cube Balls

### PR DESCRIPTION
**Pull request by Sleet827**
Help Button but now it's both Cubes and Balls

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
       <dialog id="dlg-help">
         <button autofocus title="Close" class="btn-cube btn-close"></button>
         <div class="content">
-          <p>Make 3D builds out of little cubes, you can change cube colors, view from a different angle, and save/share your build!</p>
+          <p>Make 3D builds out of little <span id="helpBtnCubeBallz">cubes</span>, you can change cube colors, view from a different angle, and save/share your build!</p>
```
Comment (Tech-Tac) [inline index.html]: Unprofessional id naming

File: index.js [hunk 2]
```diff
@@ -351,13 +351,15 @@ function toggleBalls() {
...
     document.title = "Cubes";
+    document.getElementById("helpBtnCubeBallz").innerHTML = "cubes";
   } else {
     let newStyle = document.createElement("link");
     newStyle.id = "ballsStyle";
     newStyle.rel = "stylesheet";
     newStyle.href = "balls.css";
     document.head.appendChild(newStyle);
     document.title = "Balls";
+    document.getElementById("helpBtnCubeBallz").innerHTML = "balls";
```
Comment (Tech-Tac) [inline index.js]: Same goes here

**Review by Tech-Tac — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Sleet827) [reply index.html]: nerd

File: index.js (hunk 2 above)
Comment (Sleet827) [reply index.js]: nerd

**Comment by Tech-Tac** [comment]
done

**Pull request merged**